### PR TITLE
[CDAP-13571] modify runs list title in dashboard

### PR DESCRIPTION
**Pull request by elfenheart**
JIRA: https://issues.cask.co/browse/CDAP-13571

### CODE REVIEW

File: cdap-ui/app/cdap/components/OpsDashboard/RunsGraph/DataParser.js [hunk 1]
```diff
@@ -43,7 +43,7 @@ export function parseDashboardData(rawData, startTime, duration, pipeline, custo
 
     if (buckets[startTime]) {
       // add start method
-      if (runInfo.startMethod === 'manual') {
+      if (runInfo.startMethod === 'MANUAL') {
```
Comment (bdmogal) [inline cdap-ui/app/cdap/components/OpsDashboard/RunsGraph/DataParser.js:46]: ogosh. did this change recently?

**Review by ajainarayanan — APPROVED** [review]
👍 LGTM

File: cdap-ui/app/cdap/components/OpsDashboard/RunsGraph/DataParser.js (hunk 1 above)
Comment (elfenheart) [reply cdap-ui/app/cdap/components/OpsDashboard/RunsGraph/DataParser.js:46]: yes

**Comment by elfenheart** [comment]
Build: https://builds.cask.co/browse/CDAP-URUT31

**Pull request merged**